src/api/handler.clj

```diff
@@ -22,31 +18,26 @@
 
 (def^:private app-routes
   (compojure/routes
-    (compojure/POST "/lobby" [] (api/create-game))
+    (compojure/POST "/lobby" [] {:body (api/create-game)})
     (compojure/POST "/games/:id{[0-9]+}"
                     [id]
-                    (api/add-player (parse-int id)))
+                    {:body (api/add-player (parse-int id))})
     (compojure/GET "/games/:id{[0-9]+}/player/:player"
                    [id player]
-                   (api/get-game (parse-int id) player))
+                   {:body (api/get-game (parse-int id) player)})
     (compojure/POST "/games/:id{[0-9]+}/player/:player"
                     [id player :as {body :body}]
-                    (play-action (parse-int id) player body))
-    (route/not-found "<h1>Page not found</h1>")))
+                    {:body (play-action (parse-int id) player body)})
+    (route/not-found {:body {:error "Page not found"}})))
 
-(def ^:private cors-headers 
```

Comment:
removing CORS compliance will give us problems for a Browser-Based CLI :cry:

Reply:
I'll add them again.